contracts/ETO/ETOCommitment.sol

```diff
@@ -554,7 +566,8 @@ contract ETOCommitment is
     {
         // add 1 to MIN_TICKET_TOKEN because it was produced by floor and check only MAX CAP
         // WHITELIST CAP will not induce state transition as fixed slots should be able to invest till the end of Whitelist
-        bool capExceeded = isCapExceeded(false, MIN_TICKET_TOKENS + 1, 0);
+        // also put the minimum ticket size plus one cent as eur equivalent to see wether we would cross the threshold
+        bool capExceeded = isCapExceeded(false, MIN_TICKET_TOKENS + 1, 0, ETO_TERMS.MIN_TICKET_EUR_ULPS());
```

Comment:
@rudolfix I don't really get why I don't need to add one cent to the min ticket eur ulps here? I think I should.

Reply:
done